experimental/PropertyDDS/packages/property-properties/src/properties/setProperty.js

```diff
@@ -61,10 +61,10 @@ export class SetProperty extends IndexedCollectionBaseProperty {
 	 *
 	 * ```json
 	 * {
-	 *   position: {
-	 *    x: 2,
-	 *    y: 5
-	 *   }
+	 * position: {
```

Comment:
Did eslint complain about this? It's in a code block, so I would expect it to be okay with custom indentation (assuming that was the issue).

Reply:
It was the autofixer from [jsdoc/check-alignment](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/check-alignment.md#readme). Note that these are JS files, not TS, so maybe the ruleset is slightly different?

Reply:
We'll revert it, right?

Reply:
Rule is now a warning, but it may need to be disabled altogether because otherwise the autofixer still gets triggered.